app/smc/prj.conf

```diff
@@ -58,3 +58,7 @@ CONFIG_RETENTION_BOOTLOADER_INFO_TYPE_MCUBOOT=y
 # Generate unsigned images by default
 # To sign FW, use SB_CONFIG_BOOT_SIGNATURE_KEY_FILE="/path/to/key.pem"
 CONFIG_MCUBOOT_GENERATE_UNSIGNED_IMAGE=y
+
+# DMA
+CONFIG_DMA=y
+CONFIG_DMA_ARC_HS=y
```

Comment:
I think this still needs to be removed. We should definitely have 1 pr where we're adding the driver in a test, and then another PR where we enable the driver in `app/smc` and in devicetree.
```suggestion
```